### ISSUE TITLE
Shell command execution

### ISSUE THREAD

**Issue by MoskalykA** [1148118271/ssh-rs#30]
Shell command execution does not work when my command takes longer than 500 milliseconds if the timeout is 500 milliseconds

**Comment by 1148118271** [comment]
Yes, there will be such a problem, so you need to set a reasonable timeout.

**Comment by MoskalykA** [comment]
My orders are not predictable :/

**Comment by 1148118271** [comment]
I think I probably understand what you mean, this is indeed a problem, and a more reasonable timeout verification method needs to be redesigned. Thank you very much for your feedback.

**Comment by HsuJv** [comment]
Would it be appreciatable to add a **GLOBAL** `wait infinitely` or tune the timeout from the global one to per `channel` ones, and also add `wait infinitely` to 'em

**Comment by HsuJv** [comment]
Hi @MoskalykA ,

Is this [interactive shell](https://github.com/HsuJv/ssh-rs/blob/ver_0.3/examples/shell_interactive/src/main.rs) example what you want?

BRs.
